### ISSUE TITLE
The suffix + of the option argument of commad line option -I doesn't work.

### ISSUE THREAD

**Issue by tefbop** [universal-ctags/ctags#1134]
I was trying to create a tag file for /usr/include/c++/5/future, but the parsing stopped at the macro `_GLIBCXX_VISIBILITY(default)`. I've tried using `-I _GLIBCXX_VISIBILITY+` to ignore the macro, but in vain. Manually removing the function-like macro invocation does work though.

**Comment by masatake** [comment]
Thank you for reporting.
Here is a simplified one.

```
namespace foo _GLIBCXX_VISIBILITY(default)
{
  int x;
}
```

Though I specified `-I  _GLIBCXX_VISIBILITY`, both `foo` and `x` are not captured.

**Comment by masatake** [comment]
In cxxParserParseNextToken of  `parsers/cxx/cxx_parser_tokenizer.c`:
```
	// FIXME: Handle ignore parens!
```

What @tefbop reported is about this FIXME.
That means this bug report should be recorded as a `.b` test case. I will do it.

**Comment by pragmaware** [comment]
Fixed by #1151.
